### PR TITLE
feat: use environs to get boolean and int config

### PR DESCRIPTION
**Pull request by patheard**
# Summary
Update to use the environs module for loading boolean and
integer config values.  This will provide more consistent
configuration loading behaviour.

# Related
* cds-snc/notification-planning#503

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -38,8 +41,8 @@ class Config(object):
     BULK_SEND_TEST_SERVICE_ID = os.getenv("BULK_SEND_TEST_SERVICE_ID")
     CHECK_PROXY_HEADER = False
     CONTACT_EMAIL = os.environ.get("CONTACT_EMAIL", "assistance+notification@cds-snc.ca")
-    CSV_MAX_ROWS = os.getenv("CSV_MAX_ROWS", 50_000)
-    CSV_MAX_ROWS_BULK_SEND = os.getenv("CSV_MAX_ROWS_BULK_SEND", 100_000)
+    CSV_MAX_ROWS = env.int("CSV_MAX_ROWS", 50_000)
+    CSV_MAX_ROWS_BULK_SEND = env.int("CSV_MAX_ROWS_BULK_SEND", 100_000)
```
Comment (jimleroyer) [inline app/config.py:45]: Génial

File: app/config.py [hunk 2]
```diff
@@ -87,7 +90,7 @@ class Config(object):
     PERMANENT_SESSION_LIFETIME = 20 * 60 * 60  # 20 hours
 
     REDIS_URL = os.environ.get("REDIS_URL")
-    REDIS_ENABLED = os.environ.get("REDIS_ENABLED") == "1"
+    REDIS_ENABLED = env.bool("REDIS_ENABLED", False)
```
Comment (jimleroyer) [inline app/config.py:93]: Super. Va mourrir mauvaise "1" configuration.

File: requirements.txt [hunk 3]
```diff
@@ -88,7 +92,8 @@ six==1.16.0
 smartypants==2.0.1
 statsd==3.3.0
 texttable==1.6.4
-urllib3==1.26.7
+urllib3==1.26.9
 webencodings==0.5.1
+wrapt==1.14.0
```
Comment (jimleroyer) [inline requirements.txt:97]: Pinning a dependency?

**Review by jimleroyer — APPROVED** [review]
LGTM

File: requirements.txt (hunk 3 above)
Comment (patheard) [reply requirements.txt:97]: Yeah, it got pinned by `make freeze-requirements`

**Pull request merged**